Dockerfile

```
@@ -14,4 +14,4 @@ RUN pip3 install tensorflow
 COPY . ./
 
 # Add entry point to run the script
-ENTRYPOINT [ "python3", "./analyze.py" ]
\ No newline at end of file
+ENTRYPOINT [ "python3", "./analyze.py" ]
```

Instead of a build flag, you could split ENTRYPOINT and COMMAND, so that at runtime users could override the command/args when running the container.

```suggestion
ENTRYPOINT [ "python3" ]
CMD [ "analyze.py" ]
```

---

Sure, makes sense.